.github/workflows/move_crd_values_to_operator.yaml

```diff
@@ -0,0 +1,97 @@
+name: Move Operator CRD Manifest File To Operator
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  check-specific-file:
+    runs-on: ubuntu-latest
+    outputs:
+      CHANGED: ${{ steps.check_changes.outputs.CHANGED }}
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get Source Branch Name
+        id: get_source_branch
+        run: |
+          if [[ $GITHUB_EVENT_NAME == "pull_request" ]]; then
+            echo "SOURCE_BRANCH=${{ github.head_ref }}" >> $GITHUB_ENV
+          elif [[ $GITHUB_EVENT_NAME == "push" ]]; then
+            MERGE_COMMIT=$(git log -1 --merges --pretty=%H)
+            SOURCE_BRANCH=$(git show -s --pretty=%B $MERGE_COMMIT | grep -oP '(?<=from )\S+' | sed 's/^[^/]*\///')
+            echo "SOURCE_BRANCH=${SOURCE_BRANCH}" >> $GITHUB_ENV
+          fi
+
+          if [[ "$SOURCE_BRANCH" == *operator-release* ]]; then
+            echo "Branch is a release 'release', exiting pipeline."
+            exit 0
+          fi            
+
+      - name: Check for Changes in components/tyk-operator/templates/all.yaml
+        id: check_changes
+        run: |
+          echo "Checking for changes between in ${{ env.SOURCE_BRANCH }} branch"
+          git fetch origin ${{ env.SOURCE_BRANCH }}
+          if git --no-pager log -n 1 origin/${{ env.SOURCE_BRANCH }} --name-only --pretty=format: | grep -q "components/tyk-operator/templates/all.yaml"; then
+            echo "CHANGED=true" >> $GITHUB_OUTPUT
+          else
+            echo "CHANGED=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Copy Manifest file into a temporary directory
+        if: ${{ steps.check_changes.outputs.CHANGED == 'true' }}
+        run: |
+          mkdir -p /tmp/helm
+          cp -r "${GITHUB_WORKSPACE}/components/tyk-operator/templates/all.yaml" /tmp/helm/charts-all.yaml
```

Comment:
Besides template file, there might be changes in values.yaml and Chart.yaml file.
You should commit them too.